src/amber/server/configuration.cr

```diff
@@ -22,10 +22,6 @@ module Amber
         Settings
       end
 
-      def self.key_generator
-        settings.key_generator
-      end
-
       def self.secret_key_base
```

Comment:
You can get rid of this method if you are using `Amber::Server.settings.secret_key_base`. This is a helper method for `Amber::Server.secret_key_base`

Reply:
aaah, ok.  I will use the helper method since its shorter.

Reply:
👍🏻